flips/20210824-fcl-tx-payer-service.md

```diff
@@ -0,0 +1,331 @@
+payer for a given transaction. It would exist as a resusable codebase / set of
+packages & tools that could be used by the applications that desire it.
+
+## Motivation
+
+One of the features of Flow is that transaction fees can be paid by an account
+separate from the authorizers of that transaction. Applications that wish to 
+provide "free" transactions for their users could opt to pay for their users' 
+transactions themselves. They would do this by producing a signature for the 
+transaction as the payer for that transaction. Upon receiving this transaction, 
+the Flow network would then deduct transaction fees from the payer, not any of 
+the authorizers.
+
+Some FCL compatible wallets choose to act as the payer of the transactions their users
+sign. They do this by swapping out their users account with an account of their own,
+if their users are specified as the payer of a transaction. This allows wallets with
+this feature to provide "free" transactions for their users.
+
+While this makes for a desirable user experience for those that use
+wallets with this feature, applications do not get the guarantee that _all_ of their
+users will get this user experience. This is because users are able to use whichever FCL
+compatible wallet they desire with the FCL compatible applications they use, and not
+all FCL compatible wallets will strictly choose to provide this feature.
+
+Applications that wish to guarantee their users can submit transactions
+without having to pay transaction fees for them will then have to pay those fees themselves. 
+To pay transaction fees, it's expected that applications would need to build a Transaction Payer Service that is able
+to receive a transaction signable and then produce a signature for it.
+
+A re-usable Transaction Payer Service would exist as project / set of packages and tools that applications could use
+instead of having to build an implementation of them themselves. These re-usable components
+would lower the overall complexity and effort required of a developer who wishes that 
+their application provide "free" transactions for their users.
+
+## User Benefit
+
+Most consumer applications that we interact with on a daily basis charge us nothing.
+As internet users, we're freely able to interact for our favourite social media apps,
+interact with our favourite websites and simply do as we please without paying for anything.
+However, with blockchain applications, each one of those interactions would likely be
+transaction, and with that, transaction fees would have to be paid.
+
+Users have been conditioned to expect things for free. As such, applications that wish
+to provide the best user experiences will likely choose to allow their users to execute
+transactions without them having to pay. 
+
+This Transaction Fee Payer Service will enable more applications to be more easily able to provide
+a higher level of user experience.
```

Comment:
'This Transaction Fee Payer Service will enable more applications to offer a top of the line user experience, where users can easily and seamlessly engage in any action without friction.'